unitary/examples/quantum_checkers/players.py

```diff
@@ -0,0 +1,48 @@
+from random import randint
+import traceback
+
+
+class bot:
```

Comment:
This should be capitalized "Bot", but I would delete it until it is implemented.